### PR TITLE
Cover CAPZ with hypershift e2e

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml [hunk 1]
```diff
@@ -71,6 +79,11 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure
+- as: e2e-hypershift
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: hypershift
+    workflow: hypershift-aws-e2e-external
```
Comment (damdo) [inline ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml:86]: The workflow does say aws here, is this expected?

**Review by damdo — COMMENTED** [review]
/approve

File: ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml (hunk 1 above)
Comment (enxebre) [reply ci-operator/config/openshift/cluster-api-provider-azure/openshift-cluster-api-provider-azure-master.yaml:86]: no, we'll need to setup an external for aks.
/hold

**Comment by enxebre** [comment]
/hold
a bit

**Comment by enxebre** [comment]
/close
in favour of https://github.com/openshift/release/pull/59060

**Pull request closed without merging**